example/7_allTogether.html

```diff
@@ -1,219 +1,221 @@
-  playerController.addEventListener( 'startWalking', function () { animationController.play( 'run'   ); } );
-  playerController.addEventListener( 'startJumping', function () { animationController.play( 'jump'  ); } );
-  playerController.addEventListener( 'startSliding', function () { animationController.play( 'slide' ); } );
-  playerController.addEventListener( 'startFalling', function () { animationController.play( 'slide' ); } );
+        material.skinning = true;
 
-  vent.addEventListener( 'beforerender', function () {
+      });
 
-    animationController.mesh.position.set(
-      playerController.center.x,
-      playerController.center.y - playerController.radius,
-      playerController.center.z
-    );
+      characterMesh = new THREE.SkinnedMesh(
+        characterData.geometry,
+        new THREE.MeshFaceMaterial(characterData.materials)
+      );
+      scene.add(characterMesh);
 
-    animationController.mesh.rotation.y = playerController.direction + Math.PI;
+      animationController = new MW.AnimationController(characterMesh);
+      animationController.motion.jump.setLoop(THREE.LoopOnce, 0);
+      animationController.motion.slide.setLoop(THREE.LoopOnce, 0);
+      animationController.motion.jump.clampWhenFinished = true;
+      animationController.motion.slide.clampWhenFinished = true;
 
-  } );
 
+      // player motion
+      playerController.addEventListener('startIdling', function() { animationController.play('idle'); });
+      playerController.addEventListener('startWalking', function() { animationController.play('run'); });
+      playerController.addEventListener('startJumping', function() { animationController.play('jump'); });
+      playerController.addEventListener('startSliding', function() { animationController.play('slide'); });
+      playerController.addEventListener('startFalling', function() { animationController.play('slide'); });
 
-  ;( function update () {
+      vent.addEventListener('beforerender', function() {
 
-    var delta = clock.getDelta();
+        animationController.mesh.position.set(
+          playerController.center.x,
+          playerController.center.y - playerController.radius,
+          playerController.center.z
+        );
 
-    requestAnimationFrame( update );
-    vent.dispatchEvent( { type: 'beforerender' } );
-    world.step( Math.min( delta, 0.02 ) );
-    tpsCameraControl.update();
-    animationController.update( delta );
-    renderer.render( scene, camera );
+        animationController.mesh.rotation.y = playerController.direction + Math.PI;
 
-  } )();
+      });
 
-} );
+      (function update() {
 
+        var delta = clock.getDelta();
 
+        requestAnimationFrame(update);
+        vent.dispatchEvent({ type: 'beforerender' });
+        world.step(Math.min(delta, 0.02));
+        tpsCameraControl.update();
+        animationController.update(delta);
+        renderer.render(scene, camera);
 
+      })();
```

Comment:
some spacings are also changed?
I prefer kind of [mrdoob's code style](https://github.com/mrdoob/three.js/wiki/Mr.doob's-Code-Style%E2%84%A2)